src/modules/emote_menu/utils/youtube-emotes.js

```diff
@@ -80,12 +76,15 @@ export async function loadYouTubeEmotes() {
     const categoryName = categoryNames[channelId] || DEFAULT_CATEGORY_NAME;
     const category = getCategoryForChannelId(channelId, categoryName);
     const categoryEmotes = categoryEmojis[channelId].map(
-      ({emojiId: emoteId, image, shortcuts}) =>
+      ({emojiId: emoteId, image, shortcuts, isLocked = false}) =>
         new Emote({
           id: emoteId,
           category,
           channel: categoryName,
           code: shortcuts.find((shortcut) => !shortcut.startsWith(':_')) || shortcuts[0],
+          metadata: {
+            locked: isLocked,
```

Comment:
`locked` here should be a predicate which checks if the user is subscribed or not on render.